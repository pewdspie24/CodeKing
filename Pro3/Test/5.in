100
71 90989196
932661 337069
52691 450450
416407 700018
99203 719784
727145 355207
644385 847206
710469 129349
221227 321830
668584 301537
242761 706588
242304 921331
826527 85828
98324 498318
217490 383977
82592 75185
898355 872727
401185 712946
854941 748368
224956 238060
222995 990680
185746 279319
504062 168116
291274 880828
73935 784582
221868 879134
477698 823520
785439 859998
667623 776919
964959 594854
16979 836720
513890 93022
527440 107881
872473 100155
337936 309217
497113 133459
662204 906102
641115 811003
684607 790808
440341 996502
859719 100594
184214 482721
815626 858492
516065 562041
228982 176110
482932 603622
411657 864007
693304 392587
289504 697922
755606 917833
918457 231137
258710 670599
144793 810773
160934 238113
33683 874006
569948 645338
265636 251502
365329 337752
765531 315605
82439 539644
241688 281491
823022 748274
250084 540909
534222 827478
989167 532261
288372 887535
559458 673413
157046 991464
715727 433395
128589 530936
838525 917279
411536 539055
