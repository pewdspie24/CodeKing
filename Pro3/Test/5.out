118
117
115
114
113
112
110
109
108
107
105
104
103
101
100
99
98
96
95
94
92
91
90
89
87
86
85
83
82
81
80
78
77
76
74
73
72
71
69
68
67
66
64
63
62
60
59
58
57
55
54
53
51
50
49
48
46
45
44
42
41
40
39
37
36
35
34
32
31
30
28
27
26
25
23
22
21
19
18
17
16
14
13
12
10
9
8
7
5
4
3
1
1
1
1
1
1
1
1
1
